examples/php-fpm/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+---
+version: '3.9'
+services:
+  pyroscope:
+    container_name: pyroscope
+    image: 'pyroscope/pyroscope:latest'
+    ports:
+      - '4040:4040'
+    command:
+      - 'server'
+  nginx:
+    container_name: pyroscope_nginx
+    build:
+      context: .
+      dockerfile: nginx/Dockerfile
+    ports:
+      - 80:80
```

Comment:
Opening ports under 1024 require root IIRC, may be better to use a different one?

Reply:
nginx was removed from the example